README.md

```diff
@@ -3,9 +3,9 @@
 Latin for *work*.
 
 [![Build Status](https://travis-ci.org/CS2103JAN2017-W15-B3/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-W15-B3/main)
-[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
+[![Build status](https://ci.appveyor.com/api/projects/status/12mctlo4ld7a7aqi?svg=true)](https://ci.appveyor.com/project/xbili/main)
```

Comment:
I have no idea why my commits will include these changes when I didn't stage them :/
Can someone resolve this?